generic/tests/cfg/boot_vm_in_1G_hugepage.cfg

```diff
@@ -0,0 +1,13 @@
+# Notes:
+#    Before start testing, please ensure your host OS support 1G hugepage.
+#    Please don't forget to update host kernel line to enable 1G hugepage
+#    support and ensure your host have enough memory to create guest memory.
+- boot_vm_in_1G_hugepage:
+    virt_test_type = qemu libvirt
+    type = boot
+    reboot_method = shell
+    kill_vm_on_error = yes
+    login_timeout = 240
+    reboot_count = 3
+    setup_hugepages = yes
+    extra_params += " -mem-path /mnt/hugetlbfs"
```

Comment:
mem-path should be "/mnt/kvm_hugepage" , it is uniform, as mount point "hugepage_path" is hard code in upstream.